finality-aleph/src/justification/mod.rs

```diff
@@ -55,36 +55,28 @@ pub struct JustificationNotification<Block: BlockT> {
 }
 
 #[derive(Clone)]
-pub struct JustificationHandlerConfig<B: BlockT> {
+pub struct JustificationHandlerConfig {
     /// How long should we wait when the session verifier is not yet available.
     verifier_timeout: Duration,
     /// How long should we wait for any notification.
     notification_timeout: Duration,
-    ///Distance (in amount of blocks) between the best and the block we want to request justification
-    min_allowed_delay: NumberFor<B>,
 }
 
-impl<B: BlockT> Default for JustificationHandlerConfig<B> {
+impl Default for JustificationHandlerConfig {
     fn default() -> Self {
         Self {
             verifier_timeout: Duration::from_millis(500),
-            notification_timeout: Duration::from_millis(1000),
-            min_allowed_delay: 3u32.into(),
+            notification_timeout: Duration::from_millis(800),
```

Comment:
Huh, why did you lower this?

Perhaps add some comments here about the choice of the constants, so it's clearer in the future when anything changes.

Reply:
Done.

Reply:
Huh, if this is the reasoning, do we even need any requests for non-children?

Reply:
We need it for catch-up sync.

Reply:
Eh? That shouldn't matter, since if we are catching up we are downloading already finalized blocks, right?